.eslintrc

```diff
@@ -27,5 +27,10 @@
     "@typescript-eslint/no-non-null-assertion": "off",
     "@typescript-eslint/ban-types": "off"
   },
-  "ignorePatterns": ["dist", "node_modules", "generated"]
+  "ignorePatterns": [
+    "dist",
+    "node_modules",
+    "generated",
+    "packages/create-ponder/templates/default/src/index.ts"
```

Comment:
I think it's better to do this in a new eslintrc within the packages/create-ponder dir